packages/grafana-ui/src/components/Link/TextLink.tsx

```diff
@@ -0,0 +1,118 @@
+import { css } from '@emotion/css';
+import React, { AnchorHTMLAttributes, forwardRef } from 'react';
+
+import { GrafanaTheme2, locationUtil, textUtil, ThemeTypographyVariantTypes } from '@grafana/data';
+
+import { useTheme2 } from '../../themes';
+import { IconName } from '../../types';
+import { Icon } from '../Icon/Icon';
+import { customColor, customWeight } from '../Text/utils';
+
+import { Link } from './Link';
+
+export interface Props extends AnchorHTMLAttributes<HTMLAnchorElement> {
```

Comment:
Not sure this exported interface is used anywhere but if it's needed I would suggest giving it a less generic name (e.g. `TextLinkProps`) both to help consumers and to prevent the [false positives in Levitate](https://github.com/grafana/grafana/actions/runs/5287467471/jobs/9568092743?check_suite_focus=true#step:8:1).